src/lib/DataSubscriber.cs

```diff
@@ -3154,7 +3170,7 @@ protected virtual void SynchronizeMetadata()
 
                             // Define SQL statement to insert new device record
                             string insertDeviceSql = database.ParameterizedQueryString("INSERT INTO Device(NodeID, ParentID, HistorianID, Acronym, Name, ProtocolID, FramesPerSecond, OriginalSource, AccessID, Longitude, Latitude, ContactList, ConnectionString, IsConcentrator, Enabled) " +
-                                                                                       "VALUES ({0}, {1}, {2}, {3}, {4}, {5}, {6}, {7}, {8}, {9}, {10}, {11}, {12}, 0, " + (SyncIndependentDevices ? "0" : "1") + ")",
+                                                                                       "VALUES ({0}, {1}, {2}, {3}, {4}, {5}, {6}, {7}, {8}, {9}, {10}, {11}, {12}, 0, " + (SyncIndependentDevices ? AutoEnableIndependentlySyncedDevices ? "1" : "0" : "1") + ")",

```

Comment:
When inserting a device for the first time, isn't it always added as a child device of the parent subscriber? What difference does it make if the child device is enabled?

Reply:
Independently synchronized devices are not children of the parent

Reply:
Hence the "independent" label

Reply:
The original thinking was to allow a full sync, then the user picking which devices they wanted to enable in the synchronized copy, like a prod to dev, for example

Reply:
Ahh, you're right. I see it now.
```c#
SyncIndependentDevices ? DBNull.Value : parentID
```